sourcegraph/learn
# Add next in series link

### What should this PR do?
Resolves [DEVED-266](https://sourcegraph.atlassian.net/browse/DEVED-266) by adding a next in series link to tutorials that are in a series.

### Why are we making this change?
- We want readers to check out more content, and a callout at the bottom of a tutorial in a series is a great way to do that.

### What are the acceptance criteria? 
- Tutorials that are in a series should have a callout box with a link to the next tutorial in the series unless they are the last tutorial in the series.
- Tutorials that are last in the series should not have a callout box displayed.
- Styles look reasonable on desktop and mobile.
- Links to the next tutorial work.

### How should this PR be tested?
- Check out the branch, navigate to a tutorial in a series, and test that the above criteria are met.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
